saga-distribution/src/release/LICENSE

```diff
@@ -368,51 +351,33 @@ io.grpc:grpc-netty (https://github.com/grpc/grpc-java) io.grpc:grpc-netty:jar:1.
 io.grpc:grpc-stub (https://github.com/grpc/grpc-java) io.grpc:grpc-stub:jar:1.11.0
-Jackson-annotations (http://github.com/FasterXML/jackson) com.fasterxml.jackson.core:jackson-annotations:bundle:2.8.7
-Jackson-core (https://github.com/FasterXML/jackson-core) com.fasterxml.jackson.core:jackson-core:bundle:2.8.7
-jackson-databind (http://github.com/FasterXML/jackson) com.fasterxml.jackson.core:jackson-databind:bundle:2.8.7
 JBoss Logging 3 (http://www.jboss.org) org.jboss.logging:jboss-logging:jar:3.3.0.Final
+Javassist (http://www.javassist.org/) org.javassist:javassist:bundle:3.20.0-GA
 Log4j Implemented Over SLF4J (http://www.slf4j.org) org.slf4j:log4j-over-slf4j:jar:1.7.25
+Netty/Netty (https://github.com/netty/netty) io.netty:netty:jar:3.2.5.Final
 Netty/Buffer (http://netty.io/netty-buffer/) io.netty:netty-buffer:jar:4.1.21.Final
 Netty/Codec/HTTP2 (http://netty.io/netty-codec-http2/) io.netty:netty-codec-http2:jar:4.1.21.Final
 Netty/Codec/HTTP (http://netty.io/netty-codec-http/) io.netty:netty-codec-http:jar:4.1.21.Final
 Netty/Codec (http://netty.io/netty-codec/) io.netty:netty-codec:jar:4.1.21.Final
-Netty/Codec/Socks (http://netty.io/netty-codec-socks/) io.netty:netty-codec-socks:jar:4.1.21.Final
-Netty/Common (http://netty.io/netty-common/) io.netty:netty-common:jar:4.1.21.Final
-Netty/Handler (http://netty.io/netty-handler/) io.netty:netty-handler:jar:4.1.21.Final
-Netty/Handler/Proxy (http://netty.io/netty-handler-proxy/) io.netty:netty-handler-proxy:jar:4.1.21.Final
-Netty/Resolver (http://netty.io/netty-resolver/) io.netty:netty-resolver:jar:4.1.21.Final
+Netty/Codec/Socks (http://netty.io/netty-codec-socks/) io.netty:netty-codec-socks:jar:4.1.22.Final
+Netty/Common (http://netty.io/netty-common/) io.netty:netty-common:jar:4.1.22.Final
+Netty/Handler (http://netty.io/netty-handler/) io.netty:netty-handler:jar:4.1.22.Final
+Netty/Handler/Proxy (http://netty.io/netty-handler-proxy/) io.netty:netty-handler-proxy:jar:4.1.22.Final
+Netty/Resolver (http://netty.io/netty-resolver/) io.netty:netty-resolver:jar:4.1.22.Final
 Netty/TomcatNative (https://github.com/netty/netty-tcnative) netty-tcnative-boringssl-static:2.0.7.Final
-Netty/Transport (http://netty.io/netty-transport/) io.netty:netty-transport:jar:4.1.21.Final
+Netty/Transport (http://netty.io/netty-transport/) io.netty:netty-transport:jar:4.1.22.Final
 Objenesis (http://objenesis.org) org.objenesis:objenesis:jar:2.5.1
 OpenCensus (https://github.com/census-instrumentation/opencensus-java) io.opencensus:opencensus-api:jar:0.11.0
 OpenCensus (https://github.com/census-instrumentation/opencensus-java) io.opencensus:opencensus-contrib-grpc-metrics:jar:0.11.0
 SnakeYAML (http://www.snakeyaml.org) org.yaml:snakeyaml:bundle:1.17
 Spring AOP (https://github.com/spring-projects/spring-framework) org.springframework:spring-aop:jar:4.3.17.RELEASE
-Spring Aspects (https://github.com/spring-projects/spring-framework) org.springframework:spring-aspects:jar:4.3.17.RELEASE
```

Comment:
Spring Boot is used by alpha-server. They are included in the alpha-server fat jar.